pkg/api/rules-schema.json

```diff
@@ -192,6 +228,18 @@
           "items": {
             "$ref": "#/definitions/backend"
           }
+        },
+        "tcp_connect_timeout": {
+          "description": "Connect/read/write timeout when talking to a route. Default is 15. Unit is in seconds.",
```

Comment:
http_req_timeout
Timeout for a HTTP request. Includes retries as well. Unit is in floating point seconds. Default 15.0s